charts/rstudio-pm/values.yaml

```diff
@@ -100,6 +100,18 @@ pod:
   serviceAccountName: false
   # -- annotations is a map of keys / values that will be added as annotations to the pods
   annotations: {}
+  # -- Additional labels to add to the rstudio-pm pods
+  labels: {}
+  # -- An array of containers that will be run alongside the main pod
+  sidecar: false
```

Comment:
I'm pretty sure the empty array `[]` is falsy in Go templates, so can we use that instead of the (confusing) `false` here?